docs/json-schema.md

```diff
@@ -857,7 +857,7 @@ JSON Schema specification defines several metadata keywords that describe the sc
 
 - `title` and `description`: information about the data represented by that schema
 - `$comment`: information for developers. With option `$comment` Ajv logs or passes the comment string to the user-supplied function. See [Options](./api.md#options).
-- `default`: a default value of the data instance, see [Assigning defaults](#assigning-defaults).
+- `default`: a default value of the data instance, see [Assigning defaults](./guide/modifying-data.html#assigning-defaults).
```

Comment:
it should be `modifying-data.md` here, vuepress will change it to html, and it would also work in md files.